lldb/include/lldb/API/SBTarget.h

```diff
@@ -184,6 +184,7 @@ class LLDB_API SBTarget {
 
   SBProcess LoadCore(const char *core_file);
   SBProcess LoadCore(const char *core_file, lldb::SBError &error);
+  SBProcess LoadCore(SBFile &file, lldb::SBError &error);
```

Comment:
Actually, can the `SBFile &` be marked `const`? This shouldn't modify the SBFile if I'm reading this correctly.